packages/amplify-nodejs-function-template-provider/src/utils/analyticsWalkthrough.ts

```diff
@@ -37,7 +37,7 @@ export async function askAnalyticsCategoryKinesisQuestions(context: any) {
         batchSize: 100,
         startingPosition: 'LATEST',
         eventSourceArn: streamArnParamRef,
-        functionTemplateName: 'trigger-kinesis.js',
+        functionTemplateName: 'trigger-kinesis.js.ejs',
```

Comment:
Does it not render with props if `.ejs` is not specified?

Reply:
It does not. The file itself was js , for the trigger flow. Hence the ejs.render() function was not getting called for this file.